drivers/disk/sdmmc_stm32.c

```diff
@@ -495,10 +495,25 @@ static void stm32_sdmmc_irq_config_func(const struct device *dev)
 	irq_enable(DT_INST_IRQN(0));
 }
 
+#if DT_INST_NODE_HAS_PROP(0, bus_width)
+#if DT_INST_PROP(0, bus_width) == 1
+#define SDMMC_BUS_WIDTH SDMMC_BUS_WIDE_1B
+#elif DT_INST_PROP(0, bus_width) == 4
+#define SDMMC_BUS_WIDTH SDMMC_BUS_WIDE_4B
+#elif DT_INST_PROP(0, bus_width) == 8
+#define SDMMC_BUS_WIDTH SDMMC_BUS_WIDE_8B
+#else
+#error "invalid SDMMC bus width"
+#endif /* DT_INST_PROP(0, bus_width) */
+#else
+#define SDMMC_BUS_WIDTH SDMMC_BUS_WIDE_1B
```

Comment:
Let's remove this and define default value in dts binding.

Reply:
done